pkg/user/handler.go

```diff
@@ -297,23 +306,34 @@ func (h Handler) RefreshToken(c *gin.Context) {
 		return
 	}
 
-	tokens, err := h.tokenService.GetTokens(user, refreshToken.ID.String())
+	rememberMe := false
+	rememberMeCookie, _ := c.Cookie("rememberMe")
+	if rememberMeCookie == "true" {
+		rememberMe = true
+	}
+
+	tokens, err := h.tokenService.GetTokens(user, refreshToken.ID.String(), rememberMe)
```

Comment:
I think we should pass in a `UUID` instead of a `string`. This way we remove a whole bunch of invalid strings from the possible inputs.

Reply:
I agree and also tried to change it but two things...

1. It's not something I introduced as part of this PR
2. The change will introduce some not so nice code when refreshing the token

Due to the above I'd appreciate keeping it out of this PR